arbnode/seq_coordinator.go

```diff
@@ -697,17 +697,17 @@ func (c *SeqCoordinator) PrepareForShutdown() {
 	parentCtx := c.StopWaiter.GetParentContext()
 	handoffCtx, cancel := context.WithTimeout(c.StopWaiter.GetContext(), c.config.ShutdownHandoffTimeout)
 	defer cancel()
-	if c.config.ShutdownHandoffTimeout != time.Duration(0) {
+	if c.CurrentlyChosen() && c.config.ShutdownHandoffTimeout != time.Duration(0) {
```

Comment:
you could put the wasChosen here (from line 712) and use that

Reply:
Done, thanks!